### PR TITLE
changelog for 0.4

### PR DESCRIPTION
**Pull request by minrk**
will release once this is in

cc @parente @willingc @JamiesHQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changes in sudospawner
 
+## 0.4
+
+0.4 adds better error handling:
+
+- report better errors when sudospawner fails, instead of `ValueError: substring not found` trying to parse the output
+- treat failures to signal the process as the process being missing, instead of raising errors
```
Comment (willingc) [inline CHANGELOG.md]: ...signal the process and log the process as being missing,...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changes in sudospawner
 
+## 0.4
+
+0.4 adds better error handling:
+
+- report better errors when sudospawner fails, instead of `ValueError: substring not found` trying to parse the output
```
Comment (willingc) [inline CHANGELOG.md]: ...fails;  change unhelpful `ValueError: substring not found` message when parsing output fails to a more helpful message and log the unparsed data.

**Comment by minrk** [comment]
@willingc thanks!

**Comment by minrk** [comment]
Released 0.4

**Pull request merged**